src/client.ts

```diff
@@ -376,8 +376,8 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
    * @returns a promise with an object with the newly created `conversationId`
    */
   async startVideoConference (roomJid: string, inviteeJid?: string): Promise<{ conversationId: string }> {
-    if (this.isGuest) {
-      throw createAndEmitSdkError.call(this, SdkErrorTypes.not_supported, 'video conferencing not supported for guests');
+    if (!this._config.jwt && !this._config.accessToken) {
```

Comment:
Why the change here?  More curious than anything but to me it _appears_ that `isGuest` should still suffice here?  `isGuest` is the result of `!this.isJwtAuth && !this._config.accessToken` and `isJwtAuth` in turn is `!!this._config.jwt`.  These look to be the same unless I am misunderstanding or missing something.

Reply:
Yeah you're right, I didn't bother looking into what `this.isGuest` was.

Reply:
Totally fair! I mean definitely a NIT but I curious more than anything and wanted to make sure I wasn't missing something